curvefs/src/client/s3/disk_cache_manager.cpp

```diff
@@ -101,6 +116,40 @@ int DiskCacheManager::Init(S3Client *client,
     return 0;
 }
 
+void DiskCacheManager::InitQosParam(bool readConf) {
+    uint64_t avgFlushIops, avgFlushBytes, burstFlushBytes,
+      burstSecs, avgReadFileIops, avgReadFileBytes;
+    if (readConf) {
+        avgFlushIops = option_.diskCacheOpt.avgFlushIops;
+        avgFlushBytes = option_.diskCacheOpt.avgFlushBytes;
+        burstFlushBytes = option_.diskCacheOpt.burstFlushBytes;
+        burstSecs = option_.diskCacheOpt.burstSecs;
+        avgReadFileIops = option_.diskCacheOpt.avgReadFileIops;
+        avgReadFileBytes = option_.diskCacheOpt.avgReadFileBytes;
+    } else {
+        avgFlushIops = FLAGS_avgFlushIops;
+        avgFlushBytes = FLAGS_avgFlushBytes;
+        burstFlushBytes = FLAGS_burstFlushBytes;
+        burstSecs = FLAGS_burstSecs;
+        avgReadFileIops = FLAGS_avgReadFileIops;
+        avgReadFileBytes = FLAGS_avgReadFileBytes;
+    }
+
+    ReadWriteThrottleParams params;
+    params.iopsWrite = ThrottleParams(
+      avgFlushIops, 0, 0);
+    params.bpsWrite = ThrottleParams(
```

Comment:
This is all read from gflag.

gflag is inited by conf when fuse process start.

Reply:
fix